src/main/java/uk/ac/cam/cl/dtg/segue/api/AdminFacade.java

```diff
@@ -1300,4 +1299,28 @@ public Response syncExternalAccounts(@Context final HttpServletRequest httpServl
                     "Fatal error while attempting to synchronise users!", e).toResponse();
         }
     }
+
+    @POST
+    @Path("/start_quartz")
+    @Produces(MediaType.APPLICATION_JSON)
+    @ApiOperation(value = "Start the Quartz Job Scheduler service if not already started.")
+    public Response resetMisuseMonitor(@Context final HttpServletRequest request) {
```

Comment:
Although it doesn't matter what a facade endpoint method name is, probably worth changing it :)

Reply:
🤦